src/bin/cybergrape/main.rs

```diff
@@ -39,21 +41,30 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     let args = GrapeArgs::parse();
 
     // logic to parse commandline arguments for serial vs binaural
-    let SAMP_RATE: f32 = args.samp_rate;
-    let UPDATE_RATE: f32 = args.update_rate;
+    let update_rate: f32 = args.update_rate;
 
     let cmd = args.command;
+
+    let num_files;
+    let _outfile;
+    let _infile_samples;
+    let infile_gains;
+    let infile_ranges;
```

Comment:
I don't really love having these all be options. @thompson-tyler can you think of a better way to handle this? Maybe we need to factor things out into smaller functions and run code based on which `cmd` case matches?